packages/commonwealth/server/routes/groups/update_group_handler.ts

```diff
@@ -56,8 +57,9 @@ export const updateGroupHandler = async (
   });
 
   // refresh memberships in background
+  // TODO: only refresh if requirements updated, not just metadata
```

Comment:
is this todo done? -- as https://github.com/hicommonwealth/commonwealth/pull/5694 only sends requirements to api if they change

Reply:
it will be!